identity/src/main/java/com/android/identity/presentationlog/PresentationLogMetadata.kt

```diff
@@ -0,0 +1,166 @@
+package com.android.identity.presentationlog
+
+import androidx.annotation.VisibleForTesting
+import co.nstant.`in`.cbor.CborBuilder
+import co.nstant.`in`.cbor.CborEncoder
+import com.android.identity.internal.Util
+import com.android.identity.util.EngagementTypeDef
+import com.android.identity.util.Timestamp
+import java.io.ByteArrayOutputStream
+
+class PresentationLogMetadata private constructor(
```

Comment:
This class's Builder is instantiated with PresentationLogEntry's Builder with ID = current time millis